docs/storage.rst

```diff
@@ -6,6 +6,17 @@ Resolwe Storage Framework is storage management system for Resolwe Flow Design.
 Currenlty it supports storing data to local filesystem, Google Cloud Storage
 and Amazon Simple Storage Service.
 
+
+Testing connectors
```

Comment:
Move documentation update to a separate commit as it is not connected to the browsable view.

Reply:
OK.